plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/variation-selector/attribute-options/style.scss

```diff
@@ -0,0 +1,48 @@
+.wc-block-add-to-cart-with-options-variation-selector-attribute-options__pills {
+	display: inline-flex;
+	gap: $grid-unit-10;
+	list-style: none;
+	padding: 0;
+	margin: 0;
+	flex-wrap: wrap;
+}
+
+.wc-block-add-to-cart-with-options-variation-selector-attribute-options__pill {
+	border: 1px solid $black;
+	border-radius: $grid-unit-40;
+	padding: 0.25em 1em;
+	text-transform: uppercase;
+	font-size: var(--wp--preset--font-size--small);
+	cursor: pointer;
+	user-select: none;
+
+	&:hover {
+		background-color: color-mix(in srgb, var(--wp--preset--color--contrast) 10%, transparent);
+	}
```

Comment:
Not a blocker, but it would be nice to avoid showing the `:hover` styles in the editor.

[Enregistrament de pantalla del 2025-03-11 13-09-10.webm](https://github.com/user-attachments/assets/db0c4e4e-0830-4848-9b88-307574db237d)